packages/webapp/src/routes/land/upload-local-land-charge.js

```diff
@@ -0,0 +1,101 @@
+import { logger } from 'defra-logging-facade'
+import { buildConfig } from '../../utils/build-upload-config.js'
+import constants from '../../utils/constants.js'
+import { uploadFiles } from '../../utils/upload.js'
+import { generatePayloadOptions, maximumFileSizeExceeded } from '../../utils/generate-payload-options.js'
+import { checkApplicantDetails, processRegistrationTask } from '../../utils/helpers.js'
+
+const localChargeId = '#localChargeId'

```

Comment:
Please replace with **const localLandChargeId = '#localLandChargeId'** and update dependencies.